docs/src/content/docs/guides/onetime-remittance-fixed-receive.mdx

```diff
@@ -0,0 +1,512 @@
+
+This call obtains an access token that allows your app to request a quote resource be created on the sender's wallet account.
+
+<Tabs syncKey="lang">
+  <TabItem label='TypeScript/JavaScript' icon='seti:javascript'>
+  ```ts wrap
+  const senderQuoteGrant = await client.grant.request(
+    {
+      url: senderWalletAddress.authServer
+    },
+    {
+      access_token: {
+        access: [
+          {
+            type: 'quote',
+            actions: ['create']
+          }
+        ]
+      }
+    }
+  )
+  ```
+  </TabItem>
+</Tabs>
+
+<details>
+<summary>Example response</summary>
+The following shows an example response from the sender's wallet provider.
+```json wrap
+{
+  "access_token": {
+    "value": "...", // access token value for quote grant
+    "manage": "https://auth.cloudninebank.example.com/token/{...}", // management uri for access token
+    "access": [
+      {
+        "type": "quote",
+        "actions": ["create"]
+      }
+    ]
+  },
+  "continue": {
+    "access_token": {
+      "value": "..." // access token for continuing the request
+    },
+  "uri": "https://auth.cloudninebank.example.com/continue/{...}" // continuation request uri
+  }
+}
+
+```
+</details>
+
+### 5. Request the creation of a quote resource
+
+Use the access token received in the previous step to call the <Badge text="POST" variant="success" /> [Create Quote API](/apis/resource-server/operations/create-quote/).
+
+This call requests that a quote resource be created on the sender's wallet account. The request must contain the `receiver`, which is the recipient's incoming payment `id`, along with the `receiveAmount`, which is the exact amount the sender wants the recipient to receive.
+
+The `receiveAmount` specifies that the recipient will receive exactly $5,000 MXN.
+
+<Tabs syncKey="lang">
+  <TabItem label='TypeScript/JavaScript' icon='seti:javascript'>
+  ```ts wrap
+  const senderQuote = await client.quote.create(
+    {
+      url: senderWalletAddress.resourceServer,
+      accessToken: senderQuoteGrant.access_token.value
+    },
+    {
+      method: 'ilp',
+      walletAddress: senderWalletAddress.id,
+      receiver: recipientIncomingPayment.id,
+      receiveAmount: {
+        value: '500000',
+        assetCode: 'MXN',
+        assetScale: 2
+      }
+    }
+  )
+```
+
+  </TabItem>
+</Tabs>
+
+The response returns a `receiveAmount`, a `debitAmount`, and other required information.
+
+- `debitAmount` - The amount the sender must pay (in USD in our example) after currency conversion.
+- `receiveAmount` - The amount the recipient will actually receive (exactly $5,000 MXN in our example).
+
+You'll use this same `receiveAmount` in the next step when requesting the outgoing payment grant, so the sender authorizes this exact amount.
+
+:::note[Expiring quotes]
+Quotes include an `expiresAt` timestamp. Create the outgoing payment before the quote expires. If creation fails because the quote expired, request a new quote and try again.
```

Comment:
```suggestion
Quotes include an `expiresAt` timestamp in the response. Create the outgoing payment before the quote expires. If creation fails because the quote expired, request a new quote and try again.
```

Reply:
It seems odd that we have a note, "Quotes include..." but then don't show it in the response. With the convos yesterday, I'm not sure if that was intentional.